app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -36,7 +36,8 @@
         ORDERS("orders"),
         PATIENTS("patients"),
         USERS("users"),
-        SYNC_TOKENS("sync_tokens");
+        SYNC_TOKENS("sync_tokens"),
+        UNSYNC_TOKENS("unsync_forms");
```

Comment:
@dancunningham, @capnfabs @zestyping  what do you all mean by 'Order execution is synchronous-only' ? When app tries to submit the order, if it's ok, there is nothing to do. If not, it goes to an unsent queue.
This is already synchronized, right ? It will need to wait for server response anyways. Does it make sense?

About (2), that's a really good point, @zestyping. I was doing as @capnfabs said, trying to send forms before pulling new forms. But we can add the behavior to when it tries to submit any form tom. Thx for the suggestion